installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc

```diff
@@ -28,6 +28,10 @@ include::modules/ipi-install-bmc-addressing-for-dell.adoc[leveloffset=+1]
 
 include::modules/ipi-install-bmc-addressing-for-hpe.adoc[leveloffset=+1]
 
+ifeval::[{product-version} > 4.7]
+include::modules/ipi-install-bmc-addressing-for-fujitsu.adoc[leveloffset=+2]
```

Comment:
Let's re-title the module to not list a customer names, but to make it more clear that this is for a specific HW that a customer can purchase / use.

Reply:
Renamed modules to include protocols after vendor names. Dell iDRAC, HPE iLO, Fujitsu iRMC.